gslib/tests/test_copy_helper_funcs.py

```diff
@@ -111,14 +111,15 @@ def testGetPartitionInfo(self):
     self.assertEqual(50, component_size)
 
   def testFilterExistingComponentsNonVersioned(self):
```

Comment:
Taking a closer look, too much is happening in these tests to isolate the desired behavior.
Please create a separate test for just the bug you're fixing.

Reply:
I created a separate test. Gcloud storage does not have the same bug. Do you want me to do test parity for the new test?

Reply:
If you just add it to `gsutil_test_parity.yaml` and find a relevant test for coverage, it should be fine

Reply:
Sounds good. So after this is merged I will add it to gsutil_test_parity.yaml.